grin/src/Transformations/ExtendedSyntax/Optimising/SimpleDeadParameterElimination.hs

```diff
@@ -42,28 +43,32 @@ simpleDeadParameterElimination prog@(Program exts defs) = ana builder prog where
       | Just dead <- Map.lookup name deadArgMap
       -> SAppF name (removeDead dead args)
 
-    EBind leftExp lpat@ConstTagNode{} rightExp
-      -> EBindF leftExp (pruneVal lpat) rightExp
+    -- TODO: change this
```

Comment:
To what? :)

Reply:
This is probably a "resume work from here" comment left in the code.